docs/100.yaml

```diff
@@ -343,6 +343,10 @@
   title  : Pick’s Theorem
 93:
   title  : The Birthday Problem
+  decl   : fintype.card_embedding
```

Comment:
This should probably be removed now that there is a `mathlib archive` link, right?

Reply:
I wanted to keep it this way, so that people could see where it actually comes from. The proof in `archive` is opaque almost by design :)